frontend/public/components/factory/list.tsx

```diff
@@ -289,8 +290,9 @@ export class ColHead extends React.Component<ColHeadProps> {
       currentSortOrder,
       sortField,
       sortFunc,
+      wrap,
```

Comment:
We should try to handle long headers values without needing an extra flag. This probably won't matter when we switch to PF4 tables, though.

Reply:
Removed necessary check, need to revisit long headers.